docs/sources/search/_index.md

```diff
@@ -83,3 +83,22 @@ To filter dashboard search result by a tag, complete one of the following steps:
   All tags will be shown, and when you select a tag, the dashboard search will be instantly filtered.
 
 > **Note:** When using only a keyboard, press the `tab` key and navigate to the **Filter by tag** drop-down menu, press the down arrow key `▼` to activate the menu and locate a tag, and press `Enter` to select the tag.
+
+# Command Palette
+
+Open the Command Palette by pressing `cmd+K` in macOS or `ctrl+k` in Linux/Windows. You can also click on the input placed in the navbar.
+
+![Command Palette screenshot](/static/img/docs/command-palette/CommandPalette_doc_1.png)
+
+By typing in this field, you can:
```

Comment:
These sound like some of the benefits? Consider moving this to before the "Open the Command Palette" step.